app/lib/rocketchat/methods/roomTypeToApiType.ts

```diff
@@ -0,0 +1,14 @@
+const types = {
+	c: 'channels',
+	d: 'im',
+	p: 'groups',
+	l: 'channels'
+};
+
+// TODO: refactor this
```

Comment:
We should reuse this whole file afterwards. Maybe we can reevaluate how we handle types next week